.github/workflows/deb-sanity-builds.yml

```
@@ -0,0 +1,72 @@
+name: Sanity Debian package build
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'contrib/pc-sanity/**'
```

I'm so sorry for not noticing this before! Please change this so that it only triggers on main, else any push to that directory is going to trigger the workflow. Also, if you plan to update it multiple times per day, consider using a cron trigger like the one in [this other workflow (link)](https://github.com/canonical/checkbox/actions/runs/9357219863/workflow#L5), else you are going to spam launchpad for sync requests way too often!
```suggestion
  workflow_dispatch:
  push:
    branches:
      - main
    paths:
      - 'contrib/pc-sanity/**'
```

---

Thanks for your advise, I will modified the workflow to build daily